Dockerfile

```diff
@@ -24,16 +24,18 @@ RUN export GO_WASM_DIR=$(go list -f "{{ .Dir }}" -m github.com/CosmWasm/go-cosmw
 FROM cosmwasm/go-ext-builder:0.8.2-alpine AS go-builder
 
 RUN apk add git
-# without this, build with LEDGER_ENABLED=false
-RUN apk add libusb-dev linux-headers
+# NOTE: add these to run with LEDGER_ENABLED=true
+# RUN apk add libusb-dev linux-headers
```

Comment:
There is another block like this a few lines above (line 6-7). Should we do the same there as well?

Reply:
Good call. I can remove that one entirely.
I guess I'll throw that in an existing branch - #229 to avoid a micro-pr

Reply:
https://github.com/CosmWasm/wasmd/pull/229/commits/08aef4a650afb94b4a80f64d80d185087629348e